client/src/views/Orders.vue

```diff
@@ -1,31 +1,177 @@
 <template>
   <div>
-    <h1>Orders</h1>
-    <div v-for="order in orders" :key="order.id">
-      <router-link :to="{ path: `/admin/order/${order.id}` }">
-        {{ order.id }}
-      </router-link>
+    <PageHeader>Orders</PageHeader>
+      <div class="container mx-auto my-10 md:my-12 px-4 flex flex-col md:flex-row">
```

Comment:
I think the indentation was a little off here
```suggestion
    <div class="container mx-auto my-10 md:my-12 px-4 flex flex-col md:flex-row">
```